### ISSUE TITLE
Minimum system requirements

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#25]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [x] Administrator
- [x] Developer

### Needs what

To know the minimum system requirements to run the code.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #6, #24 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
